core/dogu_format_manager.go

```diff
@@ -0,0 +1,158 @@
+package core
+
+import (
```

Comment:
This whole file misses a lot of doc comments for public symbols. It is one thing to not comment on fields, but from my point of view all structs and methods should've receive a proper doc comment in the cesapp already.

- [ ] done